plugin/storage/cassandra/schema/docker.sh

```diff
@@ -3,7 +3,7 @@
 # This script is used in the Docker image jaegertracing/jaeger-cassandra-schema
 # that allows installing Jaeger keyspace and schema without installing cqlsh.
 
-CQLSH=/opt/apache-cassandra-3.0.12/bin/cqlsh
+CQLSH=/usr/bin/cqlsh
```

Comment:
Exactly, this script is just a wrapper for `cqlsh`. I don't know why it even sets env vars like DATACENCER, MODE, I think it should delegate most of the stuff to `create.sh`